ReactNative.V8Jsi.Windows.targets

```diff
@@ -1,31 +1,24 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Set V8_ARCH to "win32" for desktop builds -->
+    <V8_ARCH Condition="'$(V8_ARCH)' == ''">uwp</V8_ARCH>

```

Comment:
(nit)

The convention for MSBuild properties is Pascal Case.

Suggestion: Call those `V8Arch`, `V8Platform`, etc.